src/intents.cs

```diff
@@ -2840,9 +3011,14 @@ interface INRideCompletionStatus : NSCopying, NSSecureCoding {
 	interface INRideDriver : NSCopying, NSSecureCoding {
 
 		[Export ("initWithPersonHandle:nameComponents:displayName:image:rating:phoneNumber:")]
-		[DesignatedInitializer]
+		[Deprecated (PlatformName.iOS, 10,2, message:"Use the designated initializer instead")]
```

Comment:
The app developer has no way of knowing which initializer is the designated one, so maybe change the message to state the overload signature instead.